main/src/cgeo/geocaching/activity/TokenAuthorizationActivity.java

```diff
@@ -0,0 +1,319 @@
+package cgeo.geocaching.activity;
```

Comment:
What exactly is this class needed for? I thought that we can use our OAuthAuthorization activity for all kinds of OAuth authorization, Geokrety included. I must be missing something here.